p4src/tna/build.sh

```diff
@@ -37,20 +39,37 @@ function base_build() {
   p4c_flags=""
   mkdir -p ${output_dir}
   (
-    time $P4C_CMD --arch tna -g --create-graphs --verbose 2 \
-      -o ${output_dir} -I ${P4_SRC_DIR} \
-      ${OTHER_PP_FLAGS} \
-      ${p4c_flags} \
-      --p4runtime-files ${output_dir}/p4info.txt \
-      --p4runtime-force-std-externs \
-      ${DIR}/fabric_tna.p4
+    if [ "${1}" == "true" ]; then
+      time $P4C_CMD --arch tna -g --create-graphs --verbose 2 \
```

Comment:
Is there a way to avoid duplication of the command? Maybe include the arguments in `$P4C_CMD`?